tests/ui/arch/all.stderr

```diff
@@ -0,0 +1,7 @@
+error: error:0:0 - Expected operand to be vector bool: All
+  |
+  = note: spirv-val failed
+  = note: module `$TEST_BUILD_DIR/arch/all.stage-id.spv.dir/module`
+
+error: aborting due to previous error
+
```

Comment:
This (and the `any` one) looks very sus, like something broke? I'll look into it.

Reply:
Okay this is broken in two different ways:
1. `// build-pass` tests should not pass in combination with "aborting due to previous error"
2. `glam` `0.22` [broke `BVec` types](https://docs.rs/glam/0.22.0/src/glam/bool/bvec2.rs.html#10-13) (lacking the `#[cfg_attr(target_arch = "spirv", repr(simd))]` they had before)

Reply:
Ugh that's annoying

Reply:
For now, to avoid this error, we can disable these:
https://github.com/EmbarkStudios/rust-gpu/blob/097ba40bedd74eeaa296e719ef7e41f2d3d76c23/crates/spirv-std/src/vector.rs#L10-L15

(since the point of these is effectively "has `#[repr(simd)]`")